### PR TITLE
Fixed the Bug in the Workshop's Show More & Show Less buttons

### LINKED ISSUE #981: [NG] see more not working properly in workshop grid in layer5-ng
**Description**
Currently, when see more button another card is clicked with the previous one being open, the previous one collapses
ezgif-2-421f984b965e

**Expected Behavior**
The previous one should collapse and the new one should expand. 

---

[Optional] **To Reproduce**

[Optional] **Additional Context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by kishore007k**
Signed-off-by: kishore007k <kishorekumarstar07@gmail.com>

**Description**

This PR fixes #981 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Workshop-grid/index.js [hunk 1]
```diff
@@ -106,10 +107,25 @@ const WorkshopsPage = ({hide_path}) => {
                                                     </a> : ""}
                                             </div>
                                             {frontmatter.status === "delivered" ? "" : <p>Upcoming...</p>}
-                                            <button type="button" className="readme-btn" onClick={() => {
-                                                setID(id);
-                                                setContent(!content);
-                                            }}>
+                                            <button type="button" className="readme-btn" onClick={open ? 
+                                                ID === id ?
```
Comment (Tanuj22) [inline src/sections/Workshop-grid/index.js:111]: Create a separate function out of this so that that it's easier to read.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
A small change and we are good to go

File: src/sections/Workshop-grid/index.js (hunk 1 above)
Comment (kishore007k) [reply src/sections/Workshop-grid/index.js:111]: I tried that But I couldn't. Can you direct me to how it can be done? @Tanuj22

Comment (Tanuj22) [reply src/sections/Workshop-grid/index.js:111]: Basically, pass `id` as an argument to the function. Something like this `onClick={()=>handleClick(id)}` (might we wrong about the exact syntax) and in the definition of the function use normal if else for doing the necessary. If you need further help ping me on slack.

**Review by Jashpatel1 — APPROVED** [review]
Nice work here. Thanks @kishore007k

**Comment by kishore007k** [comment]
Thank you @Jashpatel1

**Review by Tanuj22 — APPROVED** [review]
Thanks @kishore007k

**Comment by Tanuj22** [comment]
I wonder if a better way to do this maybe use Reducer hooks. Something to look into

**Pull request merged**